iOS/RemoteCollectionController.swift

```diff
@@ -14,6 +14,7 @@ class RemoteCollectionController: UICollectionViewController {
     lazy var sectionScrubber: SectionScrubber = {
         let scrubber = SectionScrubber(collectionView: self.collectionView!)
         scrubber.delegate = self
+        scrubber.dataSource = self
```

Comment:
Couldn't we just use the normal delegate to handle this function? I don't really see why it's a 'dataSource' it's just setting the frame right?